extensions/data-plane-transfer/data-plane-transfer-client/README.md

```diff
@@ -0,0 +1,37 @@
+# Data Plane Transfer Client extension
```

Comment:
I wonder if this documentation belongs rather in `/docs/developer' @paullatzelsperger

Reply:
Originally we had the idea of also including doc in extensions. We need to go through and rationalize the doc as a lot of it has become outdated.